project/en-US/stm/stm.xml

```diff
@@ -111,7 +177,7 @@ public interface Atomic
                 if this change is eventually supported. https://issues.jboss.org/browse/JBTM-2172</para>
         </note>
         <para>Now we have a transactional class, by virtue of its dependency on the Atomic interface, how we we go about
-            creating instances of the corresponding transactional object and use it (them) within transactions?</para>
+            creating instances of the corresponding STM object and use it (them) within transactions?</para>
```

Comment:
my comment applies to the preceding line in the source (same paragraph though). I noted an apparent existing typo "we we go"?

Reply:
I'll fix it but that was there from the start ;)